python3/httplib2/__init__.py

```diff
@@ -185,6 +185,13 @@ def _build_ssl_context(
         ssl.CERT_NONE if disable_ssl_certificate_validation else ssl.CERT_REQUIRED
     )
 
+    # SSLContext.maximum_version and SSLContext.minimum_version are python 3.7+.
```

Comment:
# SSLContext.maximum_version and SSLContext.minimum_version are Python 3.7+:
     # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.maximum_version